Common/Charting.cs

```diff
@@ -334,4 +392,21 @@ public enum ChartType
         /// Stacked series on top of each other.
         Stacked
     }
+
+    /// <summary>
+    /// Shape or symbol for the marker in a scatter plot
+    /// </summary>
+    public enum ScatterMarkerSymbol
```

Comment:
I just re-read your comment on the serialization of these values and high charts. By default JSON.NET will serialize enumerations as integer values. Place `[JsonConverter(typeof(StringEnumConverter)]` attribute at the top to use the string values of the enumerations. You can then place an `[EnumMember(Value="my-serialized-enum-value")]` on each enum member to specify the string to be used (by default will use the enum code name).